src/steps/send/poll_transaction.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  instruction:
+    "Poll /transaction endpoint until transaction status is 'pending_sender'",
+  action: "GET /transaction (SEP-0031)",
+  execute: async function(state, { request, response, instruction, expect }) {
+    const send_server = state.send_server;
+    const params = { id: state.transaction_id };
+    let transactionStatus;
+    while (transactionStatus !== "pending_sender") {
+      request("GET /transaction", params);
+      const resp = await fetch(
+        `${send_server}/transaction?` + new URLSearchParams(params).toString(),
+        {
+          headers: {
+            Authorization: `Bearer ${state.token}`,
+          },
+        },
+      );
+      expect(
+        resp.status === 200,
+        `GET /transaction responded with status ${resp.status}`,
+      );
+      let result = await resp.json();
+      response("GET /transaction", result);
+      transactionStatus = result.transaction.status;
```

Comment:
we can add a sleep in here so it doesn't just constantly spam poll

```
await new Promise((resolve) => setTimeout(resolve), 2000)
```

This might be a good utility function to add